src/pydocstyle/wordlists.py

```diff
@@ -0,0 +1,39 @@
+"""Wordlists loaded from package data.
+
+We can treat them as part of the code for the imperative mood check, and
+therefore we load them at import time, rather than on-demand.
+
+"""
+import re
+import pkgutil
+import snowballstemmer
+
+
+#: Regular expression for stripping comments from the wordlists
+COMMENT_RE = re.compile(r'\s*#.*')
+
+#: Stemmer function for stemming words in English
+stem = snowballstemmer.stemmer('english').stemWord
+
+
+def load_wordlist(name):
+    """Iterate over lines of a wordlist data file.
+
+    `name` should be the name of a package data file within the data/
+    directory.
+
+    Whitespace and #-prefixed comments are stripped from each line.
+
+    """
+    text = pkgutil.get_data('pydocstyle', 'data/' + name).decode('utf8')
+    for l in text.splitlines():
```

Comment:
I prefer one-letter variables when they have very ephemeral scope! But I take your point, and `l` is specifically forbidden by PEP8.